docs/openapi/schemas.yaml

```diff
@@ -919,6 +919,151 @@ SiteTopPageList:
   type: array
   items:
     $ref: './schemas.yaml#/SiteTopPage'
+SiteExperimentVariant:
+  type: object
+  properties:
+    name:
+      description: The name of the variant
+      type: string
+    label:
+      description: The label of the variant
+      type: string
+    split:
+      description: The split allocation of the traffic to the variant
+      type: number
+    url:
+      description: The url of the variant
+      $ref: '#/URL'
+    views:
+      description: The number of views for the variant
+      type: integer
+    interactionsCount:
+      description: The interactions count for the variant
+      type: integer
+    p_value:
+      description: The p-value of the variant for the configured metric or clicks
+      type: number
+    power:
+      description: The power of the variant for the configured metric or clicks
+      type: number
+    statsig:
+      description: The statistical significance of the variant based on the configured metric or clicks
+      type: boolean
+    metrics:
+      description: The metrics for the variant
+      type: array
+      items:
+        type: object
+        properties:
+          selector:
+            description: The css selector of the element on which the interaction was performed
+            type: string
+          value:
+            description: The value of the metric on the selector
+            type: number
+          type:
+            description: The type of the metric
+            type: string
+            enum:
+              - 'click'
+              - 'convert'
+              - 'formsubmit'
+  example:
+    name: 'challenger-1'
+    label: 'Challenger 1'
+    split: 0.5
+    url: 'https://example-site.com/foo/bar'
+    views: 1000
+    interactionsCount: 200
+    p_value: 0.04
+    power: 97.6
+    statsig: true
+    metrics:
+      - selector: '.add-to-cart'
+        value: 10
+        type: 'click'
+SiteExperiment:
+  type: object
+  properties:
+    siteId:
+      description: The site ID in uuid format
+      $ref: '#/Id'
+    experimentId:
+      description: The ID of the experiment
+      type: string
+    name:
+      description: The name of the experiment
+      type: string
+    url:
+      description: The URL of the experiment
+      $ref: '#/URL'
+    status:
+      description: The status of the experiment
+      type: string
+      enum:
+        - 'ACTIVE'
+        - 'COMPLETE'
```

Comment:
@rpapani Are there only two types? I assume there must be another type as well named `INACTIVE` (experiments which are paused but not completed)

Reply:
@ramboz should we use a `DISABLED` type as well? For full experiments I've seen customers using disabled in the manifest, in that case, experiment will not run right? Do we've the `DISABLED` in experiment metadata as well, will the customer use this type or will remove the experiment metadata?

Reply:
Looking at [experimentation documentation](https://github.com/adobe/aem-experimentation/blob/main/documentation/experiments.md), there's an `INACTIVE` status, so adding `INACTIVE` to the enum.